src/App.tsx

```diff
@@ -15,71 +19,76 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+export class App extends React.Component<{}, State> {
+  state = {
+    selectedGood: 'Jam',
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+  handleButton(relatedGood: string | null) {
+    return this.setState({ selectedGood: relatedGood });
+  }
```

Comment:
maybe consider adding a new button that will delete goods?
`this.setState({ selectedGood: '' });`

Reply:
I don't see the point of doing this, it is redundant as the existing function handles that perfectly fine

Reply:
I just thought of making it more descriptive to read code, but you may as well change `handleButton` name to something more descriptive, e.g. `toggleSelectedGood`